Chap_API_Sync_Access.tex

```diff
@@ -50,29 +185,29 @@ \subsection{\code{PMIx_Put}}
 %%%%
 \summary
 
-Post a key/value pair for distribution.
+Stage a key/value pair in prepartion for being made accessible to processes.
 
 %%%%
 \format
 
-\copySignature{PMIx_Put}{1.0}{
-pmix_status_t \\
-PMIx_Put(pmix_scope_t scope, \\
-\hspace*{9\sigspace}const pmix_key_t key, \\
-\hspace*{9\sigspace}pmix_value_t *val);
-}
+\versionMarker{1.0}
+\cspecificstart
+\begin{codepar}
+pmix_status_t
+PMIx_Put(pmix_scope_t scope,
+         const pmix_key_t key,
+         pmix_value_t *val);
+\end{codepar}
+\cspecificend
 
 \begin{arglist}
 \argin{scope}{Distribution scope of the provided value (handle)}
 \argin{key}{key (\refstruct{pmix_key_t})}
 \argin{value}{Reference to a \refstruct{pmix_value_t} structure (handle)}
 \end{arglist}
 
-\returnstart
-\begin{itemize}
-\item \refconst{PMIX_ERR_BAD_PARAM} indicating a reserved key is provided in the \refarg{key} argument.
-\end{itemize}
-\returnend
+Returns \refconst{PMIX_SUCCESS} or a negative value corresponding to a \ac{PMIx} error constant.
+If a reserved key is provided in the \refarg{key} argument then \refapi{PMIx_Put} will return \refconst{PMIX_ERR_BAD_PARAM}.
```

Comment:
This was a merge issue where we went backwards on return value changes.  Fixing in a commit shortly...